usagov/usagov-2021
# Usagov 680 slow load fix

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-680

## Description

This *should* significantly speed up the homepage load. Carousel images were not optimized like the hero banner is and each was an average size of ~ 533kb. 
The twig change applies the image style max_480w to the carousel images (which scales down each photo to 480 px). 
The config changes applies to all image styles. I used the built in drupal converter to convert all the png to webp image format. With these changes the images now average ~8.9kb in size. 
 
### To Test

- Import config changes.
- You will need to upload images for the life events pages. - Even if you already have images uploaded you will need to reupload. This is because I have changed the image styles and these are applied on upload. 
- You may need to follow homepage setup instructions - see #631 

## Checklist for the Developer



## Checklist for the Peer Reviewers
